File: app/src/main/java/com/karla/learningverbs/utils/base/BaseViewModel.java
package com.karla.learningverbs.utils.base;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class BaseViewModel extends ViewModel {

    public MutableLiveData<Boolean> loading = new MutableLiveData<>();
    public MutableLiveData<String> msgError = new MutableLiveData<>();

}

File: app/src/main/java/com/karla/learningverbs/kotlin/utils/base/BaseViewModel.kt
package com.karla.learningverbs.kotlin.utils.base

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

abstract class BaseViewModel : ViewModel() {
    @JvmField
    var loading = MutableLiveData<Boolean>()
    @JvmField
    var msgError = MutableLiveData<String>()
}